redbot/cogs/trivia/trivia.py

```diff
@@ -31,13 +31,27 @@
     {
         Optional("AUTHOR"): str,
         Optional("CONFIG"): {
-            Optional("max_score"): int,
-            Optional("timeout"): Or(int, float),
-            Optional("delay"): Or(int, float),
-            Optional("bot_plays"): bool,
-            Optional("reveal_answer"): bool,
-            Optional("payout_multiplier"): Or(int, float),
-            Optional("use_spoilers"): bool,
+            Optional("max_score"): And(
+                int, lambda n: n >= 1, error="max_score must be a positive integer"
+            ),
+            Optional("timeout"): And(
+                float, lambda n: n > 0.0, error="timeout must be a positive decimal number"
+            ),
+            Optional("delay"): And(
+                float,
+                lambda n: n > 4.0,
+                error="delay must be a positive decimal number greater than 4",
```

Comment:
- delay can also be equal to 4
- "decimal number" -> "number"
```suggestion
                lambda n: n >= 4.0,
                error="delay must be a positive number greater than or equal to 4",
```